PWGLF/Tasks/Nuspex/NucleiHistTask.cxx

```diff
@@ -1824,96 +1210,71 @@ struct NucleiHistTask {
       if (!particle.isPhysicalPrimary())
         continue;
       histTrackcuts_MC->AddBinContent(2);
+      TLorentzVector lorentzVector_particle_MC{};
 
       switch (particle.pdgCode()) {
         case +211:
           histPDG_reco->AddBinContent(1);
+          lorentzVector_particle_MC.SetPtEtaPhiM(track.pt(), track.eta(), track.phi(), constants::physics::MassPiPlus);
```

Comment:
Sooo much duplication which can be avoided by just setting values of two temporary variables (bin content and mass) and doing the actual work outside the `switch` block.